.github/workflows/ci.yml

```
@@ -85,7 +85,7 @@ jobs:
           fetch-depth: 0
 
       - name: Run Markdown Lint
-        uses: github/super-linter@v3.17.0
+        uses: github/super-linter@v3.17
```

```suggestion
        uses: github/super-linter@v3
```
I think this may have to be `@v3`. 
Well at least this is based off the example on the [super-linter repo](https://github.com/github/super-linter#example-connecting-github-action-workflow).

---

Hmm thats super weird. Oh well.